### PR TITLE
Addition to speakers list

### PR DESCRIPTION
**Pull request by MarkPieszak**
Node 4 life 🎸

### CODE REVIEW

File: locale/en/get-involved/node-speakers.md [hunk 1]
```diff
@@ -112,6 +112,16 @@ If anyone is reported as malicious or making others feel uncomfortable, they may
 - [LinkedIn](https://linkedin.com/in/tracyslee)
 - Topics - Node.js Community, Women in Tech, Diversity & Inclusion, RxJS & Node.js
 
+#### Florida 🌴
```
Comment (ZYSzys) [inline locale/en/get-involved/node-speakers.md]: Could you please remove the icon just for consistency?
```suggestion
#### Florida
```

Comment (MarkPieszak) [reply locale/en/get-involved/node-speakers.md]: Fixed! Sorry silly habit :)

**Review by ZYSzys — APPROVED** [review]

**Comment by fhemberger** [comment]
Was a bit irritated at first of a Saint Petersburg in Florida, but what do you know! 😂

**Comment by MarkPieszak** [comment]
Haha yeah when they founded the town apparently they were big fans of the real town in Russia :) 
@fhemberger

**Pull request merged**